src/clr-angular/data/datagrid/datagrid.spec.ts

```diff
@@ -44,13 +46,13 @@ export default function(): void {
 
             it("allows to manually resize the datagrid", function() {
                 const organizer: DatagridRenderOrganizer = context.getClarityProvider(DatagridRenderOrganizer);
-                let resizeDone: boolean = false;
-                organizer.done.subscribe(() => {
-                    resizeDone = true;
+                let resizeSteps = 0;
+                organizer.renderStep.subscribe(() => {
+                    resizeSteps++;
                 });
-                expect(resizeDone).toBe(false);
+                expect(resizeSteps).toBe(0);
                 context.clarityDirective.resize();
-                expect(resizeDone).toBe(true);
+                expect(resizeSteps).toBe(6);
```

Comment:
Not fond of the idea that this test is going to break every time we change the internal logic of our rendering steps, but it's an easy fix every time. So it's fine for now.